server/v2/core/store/types.go

```diff
@@ -0,0 +1,50 @@
+package store
+
+import (
+	"cosmossdk.io/core/store"
```

Comment:
Any reason we have this in server/core/store instead of directly core/store?

Reply:
Can move it to core. Don't have a preference

Reply:
Given that core is for module but as well for server (as services implementation is a server concern), I think it makes sense to have this in core directly to avoid having two core (store) packages

Reply:
moved it